### ISSUE TITLE
How do I get the print log when I use the macro at compile time ?

### ISSUE THREAD

**Issue by acodervic** [scalameta/metals#5123]
I write a macro like this:

```scala
  def debugSingleImpl(expr: Expr[Any])(using Quotes) = {
    println("now is cimpiletime")
    '{ 
      println("now is runtime")
      println("Value of " + ${ Expr(expr.show) } + " is " + $expr)
    }
  }

  inline def debugSingle(expr: Any): Unit = ${ debugSingleImpl('expr) }
```
And I use it in my code :

```scala
object Main extends App {
  debugSingle(x + 61)
}
```

When the compiled done, I can't found any print messages at the VSCode OUTPUT tab of console  .

![image](https://user-images.githubusercontent.com/57204004/230047875-f7285814-bff1-48d1-8f75-684ff60358b2.png)

Where I can found the print messages in my vscode ? Thanks for anyone to help me.

**Comment by adpi2** [comment]
@acodervic What's your build server? Bloop or sbt? You should find the information in Metals doctor.

**Comment by tgodzik** [comment]
I don't think that standard output is being forwarded from the build server. It's probably best to use plain sbt or Bloop CLI in that case.

**Comment by adpi2** [comment]
> I don't think that standard output is being forwarded from the build server. It's probably best to use plain sbt or Bloop CLI in that case.

I don't know about Bloop but sbt server should be able to forward those logs, and I would consider it an issue if it does not.

**Comment by acodervic** [comment]
> @acodervic What's your build server? Bloop or sbt? You should find the information in Metals doctor.

I build application with matels plugin for vscode .

![image](https://user-images.githubusercontent.com/57204004/230289718-5744b508-fffa-4702-8898-85495243dbdc.png)

**Comment by tgodzik** [comment]
So it seems that Bloop is eating the output from macros. You might want to try sbt BSP, you need to run tehe `Metals: switch bsp server` command (or something along these lines)

I will investigate on the Bloop side in the meantime.

**Comment by acodervic** [comment]
> 

I switched to the SBT build server , Still  haven't seen any logs of macro .

**Comment by tgodzik** [comment]
oh, that's something we should dig in then.
